### ISSUE TITLE
TCK Issue : Stub and Rest Client URL mismatch

### ISSUE THREAD

**Issue by jGauravGupta** [eclipse/microprofile-rest-client#101]
1- [AsyncMethodTest#testExecutorService](https://github.com/eclipse/microprofile-rest-client/blob/master/tck/src/main/java/org/eclipse/microprofile/rest/client/tck/asynctests/AsyncMethodTest.java#L153) method stub is created with path "**/execSvc**" and Rest Client interface "**SimpleGetApiAsync**" path is "**/**" blank which cause 404 Not Found error.

2- The same issue, I have observed with test [AsyncMethodTest#testAsyncInvocationInterceptorProvider](https://github.com/eclipse/microprofile-rest-client/blob/master/tck/src/main/java/org/eclipse/microprofile/rest/client/tck/asynctests/AsyncMethodTest.java#L200) method where stub is created for path "**/asyncIntercept**", Rest Client interface "**SimpleGetApiAsync**" final updated path (using filter) is "**/808**" and finally verification is done for "**/asyncIntercept/808**".

If it is not an issue, please share the info about, how it is expected to work behind the scene.

**Comment by andymc12** [comment]
Yes, this looks like a bug.  I don’t recall what I was originally thinking for these tests, but I was probably planning to use a different client interface, decided to use SimpleGetApiAsync instead, but forgot to change the expected path in the tests.

Would you mind fixing this - either by adding new methods (with ‘@Path’ annotations) to SimpleGetApiAsync or by changing the paths in the tests?  Otherwise I can fix it when I get back to the office tomorrow.

Thanks!

**Comment by jGauravGupta** [comment]
Thanks @andymc12 for confirmation, I will fix these tests.
